### DISCUSSION TITLE
alpinejs UI components vs bootstrap bulma etc

### DISCUSSION THREAD

**Discussion by laoshaw** [alpinejs/alpine#4324] [4. General]
I wonder, what's the benefit to use alpinejs UI instead of using those commonly available UI libraries such as bootstrap and bulma?

**Comment by ekwoka** [comment]
Well, AlpineJS UI works with AlpineJS directly.

That's it.

They are headless.

**Reply by laoshaw** [reply]
what does headless mean? how is it different from using bootstrap for example, esp some of bootstrap components have their js included, will that cause issues with alpinejs?

**Reply by ekwoka** [reply]
Headless means it doesn't include the layout or styling.

Just functionality.

It's not likely that using others would directly cause issues, no.
